threedi_schema/migrations/versions/0223_upgrade_db_inflow.py

```diff
@@ -358,6 +389,11 @@ def populate_surface_and_dry_weather_flow():
     copy_v2_data_to_dry_weather_flow(src_table)
     copy_v2_data_to_surface_map(f"{src_table}_map")
     copy_v2_data_to_dry_weather_flow_map(f"{src_table}_map")
+
+    # Check if any NULL geoms are left and remove them
+    # handle_null_geoms(src_table, 'surface')
+    # handle_null_geoms(src_table, 'dry_weather_flow')
```

Comment:
should these be commented?

Reply:
No, changed in cee0a5b4a671a731d6812d0ddef8b8bf862867d0